storage/src/persistent/commit_log.rs

```diff
@@ -105,65 +110,46 @@ impl<S: CommitLogSchema> CommitLogWithSchema<S> for CommitLogs {
         let bytes = value.encode()?;
         let offset = cl
             .append_msg(&bytes)
-            .map_err(|error| CommitLogError::AppendError { error })?;
+            .map_err(|error| CommitLogError::TezedgeCommitLogError { error })?;
 
         Ok(Location(offset, bytes.len()))
     }
 
     fn get(&self, location: &Location) -> Result<S::Value, CommitLogError> {
+        println!("Simple Get");
         let cl = self
             .cl_handle(S::name())
             .ok_or(CommitLogError::MissingCommitLog { name: S::name() })?;
         let cl = cl.read().expect("Read lock failed");
-        let msg_buf = cl
-            .read(location.0, fit_read_limit(location.1))
-            .map_err(|error| CommitLogError::ReadError {
-                error,
-                location: *location,
-            })?;
-        let bytes = msg_buf.iter().next().ok_or(CommitLogError::ReadError {
-            error: ReadError::CorruptLog,
-            location: *location,
-        })?;
-        let value = S::Value::decode(bytes.payload())?;
+        let mut msg_buf =
+            cl.read(location.0 as usize, 1)
+                .map_err(|_| CommitLogError::ReadError {
```

Comment:
@mambisi 
why did you removed error?
if something is wrong, we need to know, at least from log
```
 ReadError {
        error: ReadError,
```

Reply:
@mambisi 
and please move this `commit_log.rs` to subdir `commit_log`

Reply:
Read error was the old commit log, custom error message

Reply:
read error wasnt removed i modified it

Reply:
@mambisi
but now error is supressed, and I would like to propagete it at least as a part of CommitLogError::ReadError
```
.map_err(|_|
```

Reply:
Ok